packages/sui-cz/config.js

```diff
@@ -3,13 +3,13 @@ const readdirSync = require('fs').readdirSync
 const statSync = require('fs').statSync
 
 const basePath = process.cwd()
-const package = require(path.join(basePath, 'package.json'))
+const packageConfig = require(path.join(basePath, 'package.json')).config
```

Comment:
Not, but package, yet... No it doesn't matter, right ? I'm not changing the behavior at all, which is what matters. This was change due to linting...